ale_linters/haskell/stack_ghc.vim

```diff
@@ -4,7 +4,8 @@
 call ale#Set('haskell_stack_ghc_options', '-fno-code -v0')
 
 function! ale_linters#haskell#stack_ghc#GetCommand(buffer) abort
-    return ale#handlers#haskell#GetStackExecutable(a:buffer)
+    return ale#path#BufferCdString(a:buffer)
```

Comment:
This does not fix the issue for me.

Reply:
I'm running Stack 1.9.3 (ghc nopie) on Arch Linux, and at this commit, stack is loading project modules correctly.  What is your setup?  Also, would it be possible to share an example project where you get the bug?